src/components/WebexMediaAccess/WebexMediaAccess.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Icon,
+} from '@momentum-ui/react';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Webex Media Access component displays a prompt explaining necessary access to media.
+ *
+ * @param {object} props  Data passed to the component
+ * @param {Function} joinWithoutCamera This function is triggered by clicking the "Join without camera" button
+ * @returns {object} JSX of the component
+ */
+
+export default function WebexMediaAccess({joinWithoutCamera}) {
+  return (
+    <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-media-access`}>
+      <Icon name="icon-camera_36" />
+      <p>Allow access to camera</p>
+      <p>
+        Select&nbsp;
```

Comment:
is this because of the minifier? we should find a way to configure it so that it does not remove these spaces

Reply:
It's because the linter forces tags on a new line